N/nauty/build_tarballs.jl

```diff
@@ -3,44 +3,41 @@
 using BinaryBuilder
 
 name = "nauty"
-version = v"2.6.13" # <-- This is a lie, we're bumping from 2.6.12 to 2.6.13 to create a Julia v1.6+ release with experimental platforms
+version = v"2.8.9" 
 
 # Collection of sources required to build nauty
 sources = [
-    ArchiveSource("http://pallini.di.uniroma1.it/nauty$(version.major)$(version.minor)r12.tar.gz",
-                  "862ae0dc3656db34ede6fafdb0999f7b875b14c7ab4fedbb3da4f28291eb95dc"),
-    DirectorySource("./bundled"),
+    ArchiveSource("https://pallini.di.uniroma1.it/nauty$(version.major)_$(version.minor)_$(version.patch).tar.gz",
+		  "c97ab42bf48796a86a598bce3e9269047ca2b32c14fc23e07208a244fe52c4ee"),
+    DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/nauty*
 
-# Remove misleading libtool files 
+# Remove misleading libtool files
 rm -f ${prefix}/lib/*.la
 
-# Patch based on one from Debian: add autotools build system which creates a
-# shared library; compared to the Debian version, a bunch of things we don't
-# need have been removed (perhaps even more should be removed?)
-sed -e "s/@INJECTVER@/2.6.12/" < ../patches/autotoolization.patch > ../patches/autotoolization2.patch
-atomic_patch -p1 ../patches/autotoolization2.patch
-
-rm -f makefile*
-mkdir -p m4
-autoreconf -vi
-
 export CPPFLAGS="${CPPFLAGS} -I${prefix}/include"
 export LDFLAGS="${LDFLAGS} -L${prefix}/lib"
 
-./configure \
-    --prefix=$prefix \
-    --build=${MACHTYPE} \
-    --host=$target \
-    --enable-shared \
-    --disable-static
+atomic_patch -p1 ../patches/configure.patch
```

Comment:
I would strongly recommend to annotate all patches with a comment explaining them, so that people who update this recipe have less work in the future ("hmm this patch doesn't apply anymore. What was it about anyway? Do we still need? What would be an appropriate replacement?" -- all of that is hard to answer otherwise).

Two options for that:
1. add a comment before this line
2. change the .patch to use the format produced by `git format-patch` (i.e. put it into a commit then use `git format-patch HEAD^` to produce a patch file with the commit message in it).

Reply:
I added a comment explaining the patch.

Reply:
Thank you, great comment!

Reply:
Isn't it better to patch `configure.ac`?  `configure` can change a lot from version to version.

Reply:
You're right. The patch is only needed because a runtime check in `configure.ac` is missing a default argument, causing the check to fail during cross-compilation. I will add a patch for `configure.ac` and run `autoconf` in the build script.